actions/global_actions.jsx

```diff
@@ -405,23 +406,24 @@ export function loadDefaultLocale() {
 }
 
 let lastTimeTypingSent = 0;
-export function emitLocalUserTypingEvent(channelId, parentId) {
-    const t = Date.now();
-    const membersInChannel = ChannelStore.getStats(channelId).member_count;
-
-    const license = getLicense(getState());
-    const config = getConfig(getState());
-    if (license.IsLicensed === 'true' && config.ExperimentalTownSquareIsReadOnly === 'true') {
-        const channel = ChannelStore.getChannelById(channelId);
-        if (channel && ChannelStore.isDefault(channel)) {
-            return;
+export function emitLocalUserTypingEvent(channelId, parentPostId) {
+    const userTyping = async (actionDispatch, actionGetState) => {
+        const state = actionGetState();
+        const config = state.entities.general.config;
```

Comment:
Can you use the `getConfig` selector exported from Redux?

Reply:
Yes, makes sense :)

Reply:
I'd say it's not worth fixing this _here_ since it's a copypaste from mattermost-redux...

Reply:
Happy to defer to @jespino's best judgement here.

Reply:
Done, wasn't an effort at all :)